packages/paste-core/primitives/modal-dialog/src/index.tsx

```diff
@@ -0,0 +1,19 @@
+/**
+ * If you’re wondering why we wrapped that package into our own, we reasoned that it
+ * would be best for our consumers’ DX, with reasons such as:
+ *
+ * - If we want to migrate the underlying nuts and bolts in the future, Twilio products
+ *   that depend on this primitive would need to replace all occurrences of
+ *   `import … from ‘@reach/dialog’` to `import … from ‘@some-new/package’`.
+ *   By wrapping it in `@twilio-paste/modal-dialog-primitive`, this refactor can be avoided.
+ *   The only change would be a version bump in the package.json file.
+ * - We can more strictly enforce semver and backwards compatibility than some of
+ *   our dependencies.
+ * - We can control when to provide an update and which versions we whitelist, to
+ *   help reduce potential bugs our consumers may face.
+ * - We can control which APIs we expose. For example, we may chose to enable or
+ *   disable usage of certain undocumented APIs.
+ */
+import {DialogContent as ModalDialogContent, DialogOverlay as ModalDialogOverlay} from '@reach/dialog';
+
+export {ModalDialogContent, ModalDialogOverlay};
```

Comment:
Not exporting `Dialog` from `reach` because for it to work, we'd need to import the CSS and I don't want to do that.  Just import the two parts and style them.  This is in spirit of a primitive.

Reply:
Nice, I actually prefer the names too

Reply:
👍 on the names.

Reply:
as this is the primitive, should we name the exports 

`export {ModalDialogPrimitiveContent, ModalDialogPrimitiveOverlay};`

So that when we create the actual styled modal package we don't get name clashes?

Reply:
Fixed in https://github.com/twilio-labs/paste/pull/313/commits/e07cdec968eed6f52d308546ef1fa4e059e2f5f1